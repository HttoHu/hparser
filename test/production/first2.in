[E]:[T][E2]
    ;
[E2]:<plus>[T][E2] 
    |
    ;
[T]:[F][T2]
    ;
[T2]:<mul>[F][T2] 
    |
    ;
[F] :<lp>[E]<rp> 
    | <int>
    ;