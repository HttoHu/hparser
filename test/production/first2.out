5
2 E lp int
1 E2 plus
2 F lp int
2 T lp int
1 T2 mul